### ISSUE TITLE
Video export freezes at 9%

### ISSUE THREAD

**Issue by JLemur** [OpenShot/openshot-qt#1323]
**System Details:**
* Operating System / Distro: Linux Mint 18.3
* OpenShot Version: 2.4.1 and 2.4.1-14 (dev)
* _Please attach log files if crash_

**Issue Description and steps to reproduce:**
While trying to export a video of a slideshow that consists of 216 JPG pictures with a 2 second Fade in/Fade out transition, along with 5 mp3 files for music, OpenShot reaches 9% on the export and then does not progress any further.  I noticed that up until it gets to 9%, OpenShot uses all 4 of my CPUs at 75-100%, but then when it gets to 9% it only uses 1 CPU at ~100%.  The others are left idle.

[TerminalOutput.log](https://github.com/OpenShot/openshot-qt/files/1767453/TerminalOutput.log)

**Comment by WolfieMar** [comment]
Hmm.... I Dont Know Actually! Im a Windows 10 User. Exporting Works Fine! But I Dont Know For You. What OS Are You Using?

**Comment by JLemur** [comment]
I'm using Linux Mint 18.3, and the AppImage download from the website.  I briefly tried using it in Windows 10, but it crashed before I even started to export.  I'll try it again in Windows today.

**Comment by peanutbutterandcrackers** [comment]
Whoa. OpenShot failing on Linux Mint, again... that sucks.

@DylanC - Perhaps you can provide some insights regarding this?

**Comment by DylanC** [comment]
@peanutbutterandcrackers  -  Could be hard to reproduce. I would need a project  with 216 jpg pictures and even then this may not happen for me.

**Comment by DylanC** [comment]
@JLemur - What happens if you try the same export but removing the mp3 file first?

**Comment by JLemur** [comment]
I think the issue was that the JPEGs were too large.  A couple were over 10MB.  When I resized all of them to 1280x720, OpenShot's GUI became a lot more responsive and I was able to export the video.

**Comment by DylanC** [comment]
@JLemur - Oh right. Glad you figured it out!

**Comment by peanutbutterandcrackers** [comment]
@JLemur - Glad it is working for you now. Awesome! I hope you make awesome videos with OpenShot!

**Comment by ff55lab** [comment]
> I think the issue was that the JPEGs were too large. A couple were over 10MB. When I resized all of them to 1280x720, OpenShot's GUI became a lot more responsive and I was able to export the video.

Yes, resized the images would helped in improving the responsiveness of the APP.

I had the environment Windows 10 Home 64bit with OpenShot 2.4.2, and during exporting of video, the APP was also crashed. Until I found out that the crash was consistently happening at fixed percentage. 

Quick tip is based on the total time frame duration (example 1900 seconds) multiply the "crashed" percentage (example 50%) then the exact crashed point will be at 950 seconds within the time frame. Drag and scroll to 950 seconds to watch instantly within OpenShot to see if it crashes within the OpenShot editor.

My case, it crashed exactly at the "crashed" point (that I observed during exporting of video) where the point was between Photo and Video with Transition. So, I have made the below adjustment and it worked afterward without issue in exporting task.
1. Removed transitions between any photo with any video
2. Convert m4v file to mp4 (not sure this is the one of the issues or not)
